tools/packaging/static-build/firecracker/build-static-firecracker.sh

```diff
@@ -37,5 +37,5 @@ git fetch
 git checkout ${firecracker_version}
 sudo ./tools/devtool --unattended build --release
 
-ln -sf ./build/cargo_target/x86_64-unknown-linux-musl/release/firecracker ./firecracker-static
-ln -sf ./build/cargo_target/x86_64-unknown-linux-musl/release/jailer ./jailer-static
+ln -sf ./build/cargo_target/$(uname -m)-unknown-linux-musl/release/firecracker ./firecracker-static
+ln -sf ./build/cargo_target/$(uname -m)-unknown-linux-musl/release/jailer ./jailer-static
```

Comment:
Thiese chamges would be better to come as a different commit, still par of this very same PR.

Reply:
It is kind of minor change, I have no opinion. If it is a better kata coding style, not a problem for me.

Reply:
addressed in new update as suggested @fidencio